Makefile

```
@@ -0,0 +1,7 @@
+test: lint
+	@python3 -m unittest
+
+lint:
+	@flake8
```

```suggestion
	@autopep8 -r --aggressive --diff --exit-code .
	@flake8
```

(testing out github's suggestion feature, I don't seem to be able to suggest changes to a file that isn't already in the PR to suggest the changes to .travis.yml)